test/unit-tests/openBrowser.test.js

```diff
@@ -6,23 +6,6 @@ let { openBrowser, closeBrowser, client } = taiko;
 
 let { openBrowserArgs } = require('./test-util');
 describe('OpenBrowser', () => {
-  describe('opens browser successfully', () => {
```

Comment:
why are these tests deleted?

Reply:
since the tests were not testing anything concrete ... one was ignored and the other one was asserting on a taiko function `client` which will never be null